cmd/server.go

```diff
@@ -1,11 +1,41 @@
 package main
 
 import (
-	"github.com/otwartytransport/otwartytransport-api/internal/server"
+	"fmt"
 	"log"
+	"os"
+
+	"github.com/caarlos0/env/v6"
+	"github.com/joho/godotenv"
+	"github.com/otwartytransport/otwartytransport-api/internal/server"
 )
 
+type Config struct {
+	AppPort int `env:"APP_PORT,notEmpty,unset"`
+}
+
+func loadConfig(environment string) (Config, error) {
+	var cfg Config
```

Comment:
You could move it directly above env.Parse because it is not used before

Reply:
It's returned in line 22 with error
![image](https://user-images.githubusercontent.com/19793903/193451445-4dbb4194-cc9b-49af-8045-75dda73ae27f.png)

Reply:
I would do `return Config{}, err` because we just want to return empty struct, and it is easier to notice if we don't use some empty struct created somewhere before in the code.